src/vaadin-crud.html

```diff
@@ -125,7 +121,7 @@ <h3 slot="header">[[__editorHeader]]</h3>
        * ----------------|----------------
        * `slot=grid` | To replace the autogenerated grid with a custom one.
        * `slot=form` | To replace the autogenerated form.
-       * `slot=footer` | To add some content to the toolbar.
+       * `slot=toolbar` | To replace the toolbar content. Add an element with the attribure `new-button` for the new item action.
```

Comment:
Nit: Should not have the `slot=` prefix. The name of the slot should be enough (that’s what is used in the API docs of other components, as far as I know).